src/docs/arch-log-file.dox

```diff
@@ -26,10 +26,7 @@ the header as the first record in the log file. Writing the system record
 is part of creating the new log file.
 
 All records in the log, including the header, are subject to the minimum record
-size. Typically that is 128 bytes. That value may be changed if direct I/O for
-logging is turned on. When enabling direct I/O via \c direct_io=log to ::wiredtiger_open
-then the minimum alignment is the value specified for \c buffer_alignment.
-In all other cases it is 128 bytes.
+size. Typically that is 128 bytes.
```

Comment:
I don't think this change any more without direct IO, right? So there's no reason to hedge with "typically" any more.

Reply:
Updated here [f63a1a1](https://github.com/wiredtiger/wiredtiger/pull/10809/commits/f63a1a11f4803b30a37c6ffd8a6fb7dc17520ccc)